src/Nethermind/Nethermind.Blockchain.Test/BlockTreeTests.cs

```diff
@@ -230,7 +230,7 @@ public void Cleans_invalid_blocks_before_starting()
             tree.SuggestBlock(block2);
             tree.SuggestBlock(block3);
 
-            blockInfosDb.Set(BlockTree.DeletePointerAddressInDb, block1.Hash.Bytes);
+            blockInfosDb.Set(BlockTree.DeletePointerAddressInDb, block1.Hash.ValueKeccak);
```

Comment:
Hmm.. should be an unnecessary change. Just need to change `DbExtensions` to accept `ReadOnlySpan<byte>`.